pkg/cli/mirror/mirror.go

```diff
@@ -233,8 +233,19 @@ func (o *MirrorOptions) Run(cmd *cobra.Command, f kcmdutil.Factory) (err error)
 		// Create assocations
 		assocDir := filepath.Join(o.Dir, config.SourceDir)
 		assocs, errs := image.AssociateImageLayers(assocDir, mapping)
-		if errs != nil {
-			return errs
+
+		// Instead of returning an error, just log it.
+		skipErr := func(_ error) bool {
+			ierr := &image.ErrInvalidImage{}
+			for _, e := range errs.Errors() {
```

Comment:
Are you sure you want to iterate over `errs` returned above in line 235 and not over error passed as input to this method? If you really do want that, then I'd suggest changing these invocations, the way it currently reads is super confusing.

Reply:
I can see how looping in the anonymous function could be confusing here. Updated this to make it more straightforward. Thanks for the feedback!